code/src/components/TaskSummary.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+
+
+export const TaskSummary = () => {
+  const items = useSelector(state => state.tasks.items)
+  const leftTodo = items.filter(item => !item.complete)
```

Comment:
instead of further down using leftTodo.length, you could set that here
`const leftTodo = items.filter(item => !item.complete).length`